HW_12/task2.py

```diff
@@ -0,0 +1,19 @@
+class TenEqualSigns:
+    def __enter__(self):
+        print("==========")
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        if exc_type != None:
+            print(f"Error {exc_type.__name__} was happend with {'key' + str(i)}")
+        print("==========")
+        return 1 #  something to return ???
```

Comment:
it works) but actually it returns True
```suggestion
        return True
```

If an exception is supplied, and the method wishes to suppress the exception (i.e., prevent it from being propagated), it should return a true value. Otherwise, the exception will be processed normally upon exit from this method.